### PR TITLE
Integration test for missing config file

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Integration test for missing config file

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [x] Integration test

### CODE REVIEW

File: tests/integration/test_config.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"""Integration tests for config loader."""
+
+import pytest
+
+from ols import config
+
+
+def test_load_non_existent_config():
+    """Test how loading of non-existent config is handled."""
+    global client
```
Comment (onmete) [inline tests/integration/test_config.py:10]: `global client` is this line needed?

Comment (tisnik) [reply tests/integration/test_config.py]: actually it is not needed, fixed @onmete

**Pull request merged**